hopr/hopr-lib/src/lib.rs

```diff
@@ -1399,6 +1403,26 @@ impl Hopr {
         }
     }
 
+    pub async fn close_multiple_channels(
```

Comment:
I would expect also a function where can I say to which destinations I want to close channels.

Reply:
Yes. I thought about that. E.g. Closing multiple channels (some incoming, some outgoing initiation, and some finalization) It's possible to do so on the smart contract level. The missing part which needs some revamp is the indexer-expectation. Currently we cannot use the same txn hash to resolve multiple actions. Maybe we should have it improved in 3.1